packages/react/src/components/MDXComponents/components/art-direction/art-direction.stories.jsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright IBM Corp. 2022, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react'
+import { ArtDirection } from './art-direction'
+import desktop from '../../../../../.storybook/storybook-images/desktop.jpg'
+import mobile from '../../../../../.storybook/storybook-images/mobile.jpg'
+import tablet from '../../../../../.storybook/storybook-images/tablet.jpg'
```

Comment:
How can we do it so these images live next to its story? We'll need this for documentation page images too, so we don't have to put all images in a global images folder.